blog/EclipseFuseTooling-NewGlobalConfiguration.adoc

```diff
@@ -0,0 +1,30 @@
+
+= JBoss Fuse Tooling - Support of Global configurations
+
+:page-date: 2016-05-23
+:page-layout: blog
+:page-author: apupier
+:page-tags: [jbtis,fuse, jbosscentral]
+
+Note: This article has been posted also on http://developers.redhat.com/blog/2016/05/18/jboss-fuse-tooling-support-of-global-configurations/[Red Hat Developers Program blog]
+
+As I mentioned in an earlier post[http://developers.redhat.com/blog/2016/05/17/jboss-fuse-tooling-diagram-reworked-new-shiny-colors-and-more/], the 8.0.0.Beta2 version of JBoss Fuse Tooling is now available. Apart from the diagram tooling rework, there is yet another new, awaited feature. You can find it in the new “Configurations” editor — designed to manipulate global configurations, i.e. elements defined at the Camel context scope.

```

Comment:
Should be http://developers.redhat.com/blog/2016/05/17/jboss-fuse-tooling-diagram-reworked-new-shiny-colors-and-more/[post]